### ISSUE TITLE
read multiple files of 2DVD

### ISSUE THREAD

**Issue by godofyoga** [josephhardinee/PyDisdrometer#28]
How can we read more than one files simultaneously  and  combine data to  plot combined total  drop size distribution.

**Comment by josephhardinee** [comment]
Hey yogesh, I'm not sure I follow your question exactly. Are you wanting to combine data from different times?

**Comment by godofyoga** [comment]
import os
import glob
import numpy as np
import matplotlib.pylab as plt
import pydisdrometer as pyd
import pandas as pd
folder_path = '/home/user/anaconda/DSD/Dsd_dbl/DSD_CSV/*.csv'
new_folder_path = folder_path[:-5] + 'DSD_Out/'
files = glob.glob(folder_path)
 

if not os.path.exists(new_folder_path):
        os.makedirs(new_folder_path)
        os.chdir(new_folder_path)
        lenf = len(files)
        
ZHs,ZHc,ZHx,ZHk = [],[],[],[]
AIs,AIc,AIx,AIk = [],[],[],[]
Dvd_R,filNw,Tmin,Nw,mu,D0,Dm =[],[],[],[],[],[],[]
ZDrs,ZDrc,ZDrx,ZDrk = [],[],[],[]
KDps,KDpc,KDpx,KDpk = [],[],[],[]

for filename in files:
    
    print 'Filename is: ', filename
    reader = pyd.aux_readers.HACPL_2DVD_reader.HACPL_2DVD_dsd_reader(filename,'hacpl')
    dsd = pyd.aux_readers.HACPL_2DVD_reader.read_2dvd_dsd_hacpl(filename,'hacpl')

    dsd.calculate_dsd_parameterization()
    dvd_r = np.squeeze(reader.rain_rate) # in reader I am reading 2dvd rain rate
    dsd.calculate_RR()
# Reading multiple files  and plotting  combined plot of all files

**Comment by godofyoga** [comment]
Yes  i want to combine data and plot mean profile for monsoon days June to Sept
